Anlab.Mvc/Views/TestItems/Delete.cshtml

```diff
@@ -0,0 +1,26 @@
+@model Anlab.Core.Domain.TestItem
+
+@{
+    ViewData["Title"] = "Delete Test Item";
+}
+
+<html>
+<body>
+    <h2>Delete Test Item</h2>
```

Comment:
@laholstege I think you want to remove the `<html>, <body>, and this <H2>`  And the closing tags below.
That should all be handled by the layout page.